.travis.yml

```diff
@@ -1,7 +1,9 @@
 language: node_js
 sudo: false
+distro: bionic
 node_js:
 - '10'
+- '12'
```

Comment:
Did not know we could test on different node versions in parallel 👍 
Right now CI essentially just tests for critical build errors, but other PRs are forthcoming which will expand our testing so this will take on more importance.